src/Microsoft.AspNetCore.Http/Features/RouteValuesFeature.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Routing;
+
+namespace Microsoft.AspNetCore.Http.Features
+{
+    public class RouteValuesFeature : IRouteValuesFeature
+    {
+        private readonly HttpRequest _request;
```

Comment:
This isn’t correct. Take a look at how the other features are implemented on HttpContext and friends. The HttpContext is a wrapper around features not vice versa